cloudflare/src/endpoints/workers/create_secret.rs

```diff
@@ -0,0 +1,40 @@
+use super::WorkersSecret;
+
+use crate::framework::endpoint::{Endpoint, Method};
+
+/// Create Secret
+/// https://api.cloudflare.com/#worker-create-secret
+pub struct CreateSecret<'a> {
+    /// account id of owner of the script
+    pub account_identifier: &'a str,
+    /// the name of the script to attach the secret to
+    pub script_name: &'a str,
+    /// secert's contents
```

Comment:
s/secert/secret

Also all our comments elsewhere start with a capital, could you please change yours to be consistent? :)